CMakeLists.txt

```diff
@@ -97,5 +97,6 @@ install(
 
 install(
   PROGRAMS ${CMAKE_CURRENT_SOURCE_DIR}/bin/luatest
+           ${CMAKE_CURRENT_SOURCE_DIR}/bin/luatest-tarantool-instance
```

Comment:
I agree with @Totktonada that an instance file shouldn't be executable. Can you run it with tarantool instead of executing it?

Reply:
Yes, I can, but ...

Original luatest code in the `Server:start()` function:
```lua
    self.process = Process:start(self.command, self.args, env, {
        chdir = self.chdir,
        output_prefix = self.alias,
    })
```

I can change it to something like this:
```lua
    local args = table.copy(self.args)
    table.insert(args, 1, self.command)
    self.process = Process:start('tarantool', args, env, {
        chdir = self.chdir,
        output_prefix = self.alias,
    })
```

The problem is that I have to provide the full executable path to make [execve](https://github.com/tarantool/luatest/blob/cb28d9389249df29f4164540152185c927822607/luatest/process.lua#L69) not fail with `No such file or directory` error.

Reply:
Something like this should do the trick:

```lua
    local args = table.copy(self.args)
    table.insert(args, 1, self.command)
    self.process = Process:start(arg[-1], self.args, env, { <...> })
```

Reply:
> The problem is that I have to provide the full executable path to make [execve](https://github.com/tarantool/luatest/blob/cb28d9389249df29f4164540152185c927822607/luatest/process.lua#L69) not fail with `No such file or directory` error.

You can execute `/usr/bin/env tarantool ${ARGS}`.

Reply:
> Something like this should do the trick:
> 
> ```lua
>     local args = table.copy(self.args)
>     table.insert(args, 1, self.command)
>     self.process = Process:start(arg[-1], self.args, env, { <...> })
> ```

Excellent, thanks!

Reply:
> > The problem is that I have to provide the full executable path to make [execve](https://github.com/tarantool/luatest/blob/cb28d9389249df29f4164540152185c927822607/luatest/process.lua#L69) not fail with `No such file or directory` error.
> 
> You can execute `/usr/bin/env tarantool ${ARGS}`.

Thanks, I think @Totktonada's solution is exactly what I need here. Let's focus on it.

Reply:
Fixed.